### ISSUE TITLE
listr MissingSchema error

### ISSUE THREAD

**Issue by lparsons** [refgenie/refgenie#211]
A new installation of refgenie using biconda results in the following error when attempting `refgenie listr`:

```
$ refgenie listr
Traceback (most recent call last):
  File "/Genomics/grid/users/lparsons/miniconda3/envs/refgenie/bin/refgenie", line 10, in <module>
    sys.exit(main())
  File "/Genomics/grid/users/lparsons/miniconda3/envs/refgenie/lib/python3.8/site-packages/refgenie/refgenie.py", line 732, in main
    pfx, genomes, assets, recipes = _exec_list(rgc, args.command == LIST_REMOTE_CMD, args.genome)
  File "/Genomics/grid/users/lparsons/miniconda3/envs/refgenie/lib/python3.8/site-packages/refgenie/refgenie.py", line 553, in _exec_list
    assemblies, assets = rgc.get_remote_data_str(genome=genome)
  File "/Genomics/grid/users/lparsons/miniconda3/envs/refgenie/lib/python3.8/site-packages/refgenconf/refgenconf.py", line 490, in get_remote_data_str
    return self.listr(genome, order, get_url)
  File "/Genomics/grid/users/lparsons/miniconda3/envs/refgenie/lib/python3.8/site-packages/refgenconf/refgenconf.py", line 507, in listr
    url = get_url(url, API_ID_ASSETS)
  File "/Genomics/grid/users/lparsons/miniconda3/envs/refgenie/lib/python3.8/site-packages/refgenconf/refgenconf.py", line 475, in <lambda>
    def get_remote_data_str(self, genome=None, order=None, get_url=lambda server, id: construct_request_url(server, id)):
  File "/Genomics/grid/users/lparsons/miniconda3/envs/refgenie/lib/python3.8/site-packages/refgenconf/refgenconf.py", line 1705, in construct_request_url
    return server_url + _get_server_endpoints_mapping(server_url)[operation_id]
  File "/Genomics/grid/users/lparsons/miniconda3/envs/refgenie/lib/python3.8/site-packages/refgenconf/refgenconf.py", line 1719, in _get_server_endpoints_mapping
    json = _download_json(url + "/openapi.json")
  File "/Genomics/grid/users/lparsons/miniconda3/envs/refgenie/lib/python3.8/site-packages/refgenconf/refgenconf.py", line 1385, in _download_json
    resp = requests.get(url, params=params)
  File "/Genomics/grid/users/lparsons/miniconda3/envs/refgenie/lib/python3.8/site-packages/requests/api.py", line 76, in get
    return request('get', url, params=params, **kwargs)
  File "/Genomics/grid/users/lparsons/miniconda3/envs/refgenie/lib/python3.8/site-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/Genomics/grid/users/lparsons/miniconda3/envs/refgenie/lib/python3.8/site-packages/requests/sessions.py", line 516, in request
    prep = self.prepare_request(req)
  File "/Genomics/grid/users/lparsons/miniconda3/envs/refgenie/lib/python3.8/site-packages/requests/sessions.py", line 449, in prepare_request
    p.prepare(
  File "/Genomics/grid/users/lparsons/miniconda3/envs/refgenie/lib/python3.8/site-packages/requests/models.py", line 314, in prepare
    self.prepare_url(url, params)
  File "/Genomics/grid/users/lparsons/miniconda3/envs/refgenie/lib/python3.8/site-packages/requests/models.py", line 388, in prepare_url
    raise MissingSchema(error)
requests.exceptions.MissingSchema: Invalid URL 'h/openapi.json': No schema supplied. Perhaps you meant http://h/openapi.json?
```

**Comment by stolarczyk** [comment]
@lparsons, can you share the first few lines of you refgenie configuration file? Is is your list of servers a string rather than a list?

**Comment by lparsons** [comment]
It's the default config:
```
config_version: 0.3
genome_folder: /Genomics/biocomp/refgenie-genomes/genomes
genome_servers: ['http://refgenomes.databio.org']
genomes: null
```

Interestingly, I can get a pip installed version (0.9.3) to work, but the conda installed version (0.9.2) results in this error. Perhaps a problem with the conda package? I'm happy to look into fixing that, but not sure where to start. The pip install doesn't work easily on my server due to gcc version issues.

**Comment by stolarczyk** [comment]
Can you try upgrading both refgenie and refgenconf (dependency, configuration package) to the most recent versions available on bioconda? That would be 0.9.2 and 0.9.3.

**Comment by lparsons** [comment]
Bioconda did not have 0.9.3 of refgenie available, so I pushed that update in https://github.com/bioconda/bioconda-recipes/pull/24477. Then I updated by local installation with the updated bioconda recipe and everything seems to work! Thanks.
